pages/mes-jeunes/[jeune_id]/actions/index.tsx

```diff
@@ -34,21 +32,27 @@ import withDependance from 'utils/injectionDependances/withDependance'
 
 interface ActionsProps {
   jeune: Jeune
-  actions: ActionJeune[]
-  deleteSuccess: boolean
-  messageEnvoiGroupeSuccess?: boolean
+  actions: Action[]
   pageTitle: string
+  creationSuccess?: boolean
+  suppressionSuccess?: boolean
+  messageEnvoiGroupeSuccess?: boolean
 }
 
 function Actions({
   jeune,
   actions,
-  deleteSuccess,
+  creationSuccess,
```

Comment:
nitpick (si on considère que porté par l'index et la fonction):
```suggestion
  actionCreationSuccess,
```
idem pour la suppression